### PR TITLE
Add issue links to README.md

### PR DESCRIPTION
**Pull request by lbussell**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,3 +4,16 @@ This repo contains basic information about:
...
+
+## Issues
+
+Issues are disabled on this repo. Please use the following links to file issues on other related repos:
+
+- **[Issue with .NET runtime or core .NET libraries](https://github.com/dotnet/runtime/issues/new/choose)**
+- **[Issue with ASP.NET Core](https://github.com/dotnet/efcore/issues/new/choose)**
+- **[Issue with .NET SDK](https://github.com/dotnet/sdk/issues/new/choose)**
+- **[Issue with .NET containers](https://github.com/dotnet/dotnet-docker/issues/new/choose)**
+- **[Issue with other .NET components](https://github.com/dotnet/core/blob/master/Documentation/core-repos.md)**
```
Comment (mthalman) [inline README.md:16]:
```suggestion
- **[Issue with other .NET components](https://github.com/dotnet/core/blob/main/Documentation/core-repos.md)**
```

File: README.md [hunk 2]
```diff
@@ -4,3 +4,16 @@ This repo contains basic information about:
 
 - [Docker images produced by the .NET team](./build-info/docker)
```
Comment (MichaelSimons) [inline README.md:5]: I was thinking we would only provide links for the repos related to the build-info stored in this repo (e.g. the Docker repos).  Maybe including a link to core-repos as a general landing place for other issue.

File: README.md [hunk 3]
```diff
@@ -4,3 +4,12 @@ This repo contains basic information about:
...
 - [Docker images produced by the Go team](./build-info/microsoft/go-images)
+
+## Issues
+
+Issues are disabled on this repo. If you have any problems with .NET or the contents of this repo, file an issue in one of the following repos:
+
+- **[All .NET GitHub repos](https://github.com/dotnet/core/blob/main/Documentation/core-repos.md)**
+- **[.NET containers](https://github.com/Microsoft/Windows-Containers/issues)**
```
Comment (mthalman) [inline README.md:13]:
```suggestion
- **[.NET containers](https://github.com/dotnet/dotnet-docker/issues/new/choose)**
```

Comment (lbussell) [reply README.md:13]: Should I add this back for the other two repos then?

Comment (mthalman) [reply README.md:13]: What other two repos?

Comment (lbussell) [reply README.md]: I fixed the link. Separate question is whether we link directly to the new issue chooser. Go and MAR do not have as detailed issue choosers so I think we should leave it as repo links.

Comment (mthalman) [reply README.md]: Oh, I don't care about linking to the "issues" page or not. I was just intending to correct the repo.

**Review by mthalman — APPROVED** [review]

**Pull request merged**